Masses: solarsys.ssc, dwarfplanets.ssc, and outersys.ssc

JiliTheSpaceboy opened the pull request:
Updated mass values of objects in the following files:
- solarsys.ssc, updating all satellites except Phobos and Deimos.
- dwarfplanets.ssc / outersys.ssc, wherever a mass value is available for the object, as well as updating Pluto's mass value. Note that Dysnomia's mass is obtained assuming that there is a genuine detection.

Significant figures are set to combination of value of 'Earth mass' used by Celestia, 5.972E24 kg (credit to pedroJ on Discord server), and the mass value of the object itself. The sigfig is set high to avoid spurious digits in the program.

JiliTheSpaceboy commented:
Oh I forgot to mention this but Quaoar's rotation period has also been updated to 17 hr, preferred by [Morgado et al. (2023)](https://ui.adsabs.harvard.edu/abs/2023Natur.614..239M/abstract) and [Pereira et al. (2023)](https://ui.adsabs.harvard.edu/abs/2023A%26A...673L...4P/abstract).

SevenSpheres commented on data/outersys.ssc:
```diff
@@ -235,6 +241,7 @@ ReferencePoint "Orcus-Vanth" "Sol"
 	Mesh	"asteroid.cms"
 	Texture	"asteroid.*"
 	Radius	50  # dynamical upper limit
+	Mass	
```
Missing a value here; I'd also suggest adding comments with the source of these masses.

JiliTheSpaceboy replied:
Got it, I've uploaded updates to these. Xiangliu does not have mass value as far as I know so I simply removed it - I forgot to before.

JiliTheSpaceboy commented:
Celestia has recently received ability to calculate densities for all ellipsoid objects. This includes most objects in the three files I am working on here, but certain objects like Ceres and Mimas which use meshes will still not display densities, and have to rely on Density.

Should I also add density values from papers for other, ellipsoid objects, wherever available?

375gnu commented:
> Should I also add density values from papers for other, ellipsoid objects, wherever available?

I don't think it's highly necessary, but if you wish, you can do.

JiliTheSpaceboy commented:
> > Should I also add density values from papers for other, ellipsoid objects, wherever available?
> 
> I don't think it's highly necessary, but if you wish, you can do.

Welp, seeing that I’ve already added them, not gonna take them out now. Plus the value should be more accurate as well I think.

Now that I’ve fixed 7Sph’s request and added more stuff, I suppose it should be good now, unless someone else find good sources for objects that I still rely on the fact sheet right now…?
